firefox_ui_tests/remote/security/test_unknown_issuer.py

```diff
@@ -0,0 +1,40 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestUnknownIssuer(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://ssl-unknownissuer.mozqa.com'
+
+        self.marionette.set_context('content')
```

Comment:
I'm a bit worried here. Do we ever reset the context to `chrome`? Right now we should fail in tearDown if something is broken here. I feel you should better use the `with` expression in the test method.